types/selenium-webdriver/test/bidi/logInspector.ts

```diff
@@ -0,0 +1,37 @@
+import * as webdriver from "selenium-webdriver";
+import LogInspector from "selenium-webdriver/bidi/logInspector";
+
+const testText = 'hello!';
+
+async function TestLogInspector() {
+    const driver: webdriver.WebDriver = new webdriver.Builder().build();
+    const logInspector = await LogInspector(driver);
+    let error = '';
+    let complete = false;
+    const timeoutError = setTimeout(() => {
+        error = `Timed out waiting for log ${testText}`
+        complete = true
+    }, 1000)
+    logInspector.onLog(async ({text}) => {
+        if (text === testText) {
+            clearTimeout(timeoutError);
+            complete = true
+        }
+    })
+
+    await driver.executeScript(`console.log('${testText}')`);
+    checkForLog();
+
+    async function checkForLog() {
+        if (!complete) {
+            setTimeout(checkForLog)
+        }
+
+        await logInspector.close()
+        await driver.close()
+
+        if (error) {
+            throw new Error(error)
+        }
+    }
```

Comment:
This doesn't actually affect the typings so it doesn't _really_ matter, but I feel compelled to say that this code wouldn't work in practice. After calling `setTimeout(checkForLog)`, it closes the logInspector and driver even if `complete` is false. So this'll just shut everything down.

I'd recommend a pattern more like this:
```typescript
    await checkForLog(testText, 1000);
    await logInspector.close();
    await driver.close();
    
    async function checkForLog(expectedText: string, timeout: number) {
        const findLog = new Promise((resolve) => {
            logInspector.onLog(({ text }) => {
                if (text === expectedText) {
                    resolve();
                }
            });
        });
        const timeOut = new Promise((_, reject) => {
            setTimeout(() => { reject(new Error("Search timed out")); }, timeout);
        });
        return Promise.race([findLog, timeOut]);
    }
```

Reply:
@SupernaviX - Testing this test change now, thanks for the help

Reply:
@SupernaviX - Okay, all changes are made. Sorry for the delay and thanks for maintaining this repo!